src/locales/zh-tw.json

```diff
@@ -0,0 +1,857 @@
+{
+  "title": "繁體中文",
+  "App": {
+    "Printers": "印表機組",
+    "TopBar": {
+      "SAVE_CONFIG": "保存配置",
+      "UploadPrint": "上傳並列印",
+      "EmergencyStop": "緊急停止",
+      "Uploading": "上傳",
+      "UploadOfFileSuccessful": "上傳 {file} 成功!",
+      "CannotUploadTheFile": "無法上傳該檔案!"
+    },
+    "TopCornerMenu": {
+      "KlipperControl": "Klipper 控制",
+      "KlipperRestart": "Klipper 重啟",
+      "KlipperFirmwareRestart": "韌體重啟",
+      "ServiceControl": "服務控制",
+      "PowerDevices": "功率元件",
+      "HostControl": "主機控制",
+      "Reboot": "重啟",
+      "Shutdown": "關機"
+    },
+    "UpdateDialog": {
+      "Updating": "更新 {software}...",
+      "UpdatingDone": "更新 {software} 完成!",
+      "Recovering": "恢復 {software}...",
+      "RecoveringDone": "恢復 {software} 完成!",
+      "Empty": "空的",
+      "Close": "關閉"
+    },
+    "ThrottledStates": {
+      "HeadlineCurrentFlags": "目前標記",
+      "HeadlinePreviouslyFlags": "以前的標記",
+      "TitleUnderVoltageDetected": "檢測到電壓過低",
+      "TitleFrequencyCapped": "頻率上限",
+      "TitleCurrentlyThrottled": "目前被限制",
+      "TitleTemperatureLimitActive": "溫度限制",
+      "TitlePreviouslyUnderVolted": "先前的不足",
+      "TitlePreviouslyFrequencyCapped": "以前的頻率上限",
+      "TitlePreviouslyThrottled": "以前節流",
+      "TitlePreviouslyTemperatureLimited": "以前的溫度限制",
+      "DescriptionUnderVoltageDetected": "樹莓派電源電壓目前低於4.65V",
+      "DescriptionFrequencyCapped": "樹莓派ARM最大頻率目前限制為1.2 GHz.",
+      "DescriptionCurrentlyThrottled": "樹莓派ARM核心目前已被限制.",
+      "DescriptionTemperatureLimitActive": "rPi uC （僅 3A+/3B+）溫度目前高於軟限制（默認 60C）。",
+      "DescriptionPreviouslyUnderVolted": "自上次通電後，樹莓派電源電壓至少有一次降至4.65V以下.",
+      "DescriptionPreviouslyFrequencyCapped": "自上次通電後，樹莓派ARM的最大頻率至少一度限制在1.2 GHz.",
+      "DescriptionPreviouslyThrottled": "自上次通電, 後樹莓派ARM至少節流一次.",
+      "DescriptionPreviouslyTemperatureLimited": "rPi uC （僅 3A+/3B+）溫度自上次開機以來至少一次高於軟限制（默認 60C）。"
+    }
+  },
+  "EmergencyStopDialog": {
+    "EmergencyStop": "緊急停止",
+    "AreYouSure": "你確定嗎?",
+    "No": "否",
+    "Yes": "是"
+  },
+  "PowerDeviceChangeDialog": {
+    "TurnDeviceOn": "打開 {device}",
+    "TurnDeviceOff": "關閉 {device}",
+    "AreYouSure": "你確定嗎?",
+    "No": "否",
+    "Yes": "是"
+  },
+  "ConnectionDialog": {
+        "Connecting": "連接到 {host}",
```

Comment:
Hi, thanks for updating the translation!

The file looks good so far. I just saw two sections which need some fixing of their indentation levels: 
![1](http://puu.sh/Iqn73/3fcd2aa26d.png)
Please indent each new level by 2 spaces only.

After that is fixed, i think we are good to go to merge it! :)

Reply:
done